app/src/main/res/layout/activity_edit_profile.xml

```diff
@@ -6,29 +6,29 @@
     android:layout_height="match_parent">
 
     <Button
-        android:id="@+id/edit_buttonCancel"
+        android:id="@+id/editButtonCancel"
```

Comment:
For xml, the convention actually is underscores, so please change it back.

Java = camelCase
Xml= underscores

Reply:
ok got it. Thanks for point it out!